README.md

```diff
@@ -3,7 +3,7 @@ Quote the web — [install Chrome extension](https://chrome.google.com/webstore
 
 ---
 
-Grab quotes from any page using cmd+shift+s (on Windows: ctrl+shift+S). Then grab the embed link to produce a live HTML citation like this:
+Grab quotes from any page using Ctrl+Shift+S (on Macs: cmf+Shift+S). Then grab the embed link to produce a live HTML citation like this:
```

Comment:
I flipped this round, because I think the Ctrl keybinding is the default, and only Macs are different; the existing one made it sound the other way round, and didn't include Linux, BSD, so I had to guess what the correct keybinding was.